pkg/cmd/provisioning/gcp/create_workload_identity_pool.go

```diff
@@ -0,0 +1,75 @@
+package gcp
+
+import (
+	"context"
+	"fmt"
+	"github.com/openshift/cloud-credential-operator/pkg/gcp"
+	"github.com/pkg/errors"
+	"log"
+
+	"github.com/spf13/cobra"
+	"google.golang.org/api/iam/v1"
+)
+
+var (
+	// CreateWorkloadIdentityPoolOpts captures the options that affect creation of the workload identity pool
+	CreateWorkloadIdentityPoolOpts = options{
+		Name:    "",
+		Project: "",
+	}
+)
+
+func createWorkloadIdentityPoolCmd(cmd *cobra.Command, args []string) {
+	ctx := context.Background()
+
+	gcpClient, err := gcp.NewClient(CreateWorkloadIdentityPoolOpts.Project, nil)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	err = createWorkloadIdentityPool(ctx, gcpClient, CreateWorkloadIdentityPoolOpts.Name, CreateWorkloadIdentityPoolOpts.Project)
+	if err != nil {
+		log.Fatal(err)
+	}
+}
+
+func createWorkloadIdentityPool(ctx context.Context, client gcp.Client, name, project string) error {
+	// Ensure display name is no longer than 32 characters
+	var shortenedName string
+	if len(name) > 32 {
+		shortenedName = name[0:32]
+	} else {
+		shortenedName = name
+	}
+
+	pool := &iam.WorkloadIdentityPool{
+		Name:        name,
+		DisplayName: shortenedName,
+		Description: "A workload identity pool for an OpenShift cluster",
+		State:       "ACTIVE",
+		Disabled:    false,
+	}
+
+	_, err := client.CreateWorkloadIdentityPool(ctx, fmt.Sprintf("projects/%s/locations/global", project), shortenedName, pool)
```

Comment:
can we make a simple string template to store the magic "projects/%s/locations/global" string (with a hopefully descriptive variable name)?